README.md

```diff
@@ -17,9 +17,9 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] Strings
-[] Booleans
-[] Undefined
+[x] Strings
```

Comment:
:+1: I can see where this question might be confusing, integers are a type of number, which is a JS dataType, just FYI